assets/erc20.go

```diff
@@ -1,5 +1,32 @@
-package abis
+package assets
 
+import (
+	"math/big"
+)
+
+type Erc20Params struct {
+	Symbol      string
+	TokenName   string
+	Decimals    int
+	TotalSupply *big.Int
+}
+
+const ERC20_CAPPED_PAUSABLE_TEMPLATE = `pragma solidity ^0.5.2;
+
+import "./lib/oz/contracts/token/ERC20/ERC20Pausable.sol";
+import "./lib/oz/contracts/token/ERC20/ERC20Capped.sol";
+import "./lib/oz/contracts/token/ERC20/ERC20Detailed.sol";
+
+contract CappedToken is ERC20Detailed, ERC20Capped, ERC20Pausable {
+
+    // ------------------------------------------------------------------------
+    // Constructor
+    // ------------------------------------------------------------------------
+    constructor() 
+    ERC20Detailed("{{.TokenName}}", "{{.Symbol}}", {{.Decimals}}) 
+    ERC20Capped({{.TotalSupply}})
+    public {}
+}`
 const ERC20 = `[
```

Comment:
If these aren't going to be in an `abi` package, then they should probably have `ABI` in the name.